source/unified-test-format/schema-1.24.json

```diff
@@ -0,0 +1,1161 @@
+{
```

Comment:
I don't want to scope creep here but it's a bit of a pain to review UTF changes because with each version we create a entirely new file. What if we add a file called "schema-latest.json"? Then we could actually review the changes to schema-latest.json.

Reply:
@ShaneHarvey Good idea: https://github.com/mongodb/specifications/pull/1792